spec/models/qts_years_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe QtsYears do
+  describe ".first_eligible_year" do
+    it "returns the first year of the eligible QTS years" do
+      stub_const("QtsYears::ELIGIBLE_YEARS", ["2018-2019", "2019-2020"])
```

Comment:
On balance I don't think the stubbing in these specs is very helpful as it locks the spec to the implementation of the method and the underlying data structures, which makes the code harder to refactor. In reality, the QTS years for this policy are set in stone, so I think we can be explicit about that and simply say `expect(QtsYears.first_eligible_year).to eq "2013"`.